src/app/service/user/user.service.ts

```diff
@@ -223,6 +224,12 @@ export class UserService implements OnLogout {
     });
   }
 
+  updateLastTimeActivity() {
+    const date = new Date();
+    const currentDate = moment(date).format('yyyy-MM-DDTHH:mm:ss.SSSSSS');
```

Comment:
please move this line to a separate method.

Reply:
A separete reusable method for date converting has been created with this commit a535799dc3be146d73c955ca6b0c6f40dddd115c